content/applications/productivity/iot/devices/screen.rst

```diff
@@ -2,39 +2,42 @@
 Connect a screen
 ================
 
-In Odoo, an :abbr:`IoT (Internet of Things)` Box can be connected to a screen display. After being
-connected, the screen can be used to display a :abbr:`PoS (Point of Sale)` order to a client.
+In Odoo, an :abbr:`IoT (Internet of Things)` box can be connected to a screen display. After being
+configured, the screen can be used to display a Point of Sale (PoS) order to a client.
 
 .. image:: screen/screen-pos-client-display.png
    :align: center
    :alt: An example of a PoS (point of sale) order on a screen display.
 
-.. note::
-   Access the customer display from any other computer by going to the :abbr:`IoT (Internet of
-   Things)` Box homepage and clicking on the :guilabel:`POS Display` button.
+Access the customer display by going to the :abbr:`IoT (Internet of Things)` box homepage and
+clicking on the :guilabel:`PoS Display` button. To get to the :abbr:`IoT (Internet of Things)` box
+homepage, navigate to :menuselection:`IoT app --> IoT Boxes` and click on the :abbr:`IoT (Internet
+of Things)` box homepage link.
 
 Connection
 ==========
 
-The way to connect the screen display to the :abbr:`IoT (Internet of Things)` Box differs depending
+The way to connect the screen display to the :abbr:`IoT (Internet of Things)` box differs depending
 on the model.
 
 .. tabs::
 
    .. tab:: IoT Box model 4
 
       Connect up to two screens with Micro-HDMI cables on the side of the :abbr:`IoT (Internet of
-      Things)` Box. If two screens are connected, they can display distinct content (see usage
-      below).
+      Things)` box. If two screens are connected, they can display distinct content (see
+      :ref:`Screen Usage <iot/devices/usage_screen>`). :ref:`See the Raspberry Pi Schema
+      <iot/config/connect_schema>`.
 
    .. tab:: IoT Box model 3
 
-      Connect the screen with an HDMI cable on the side of the :abbr:`IoT (Internet of Things)` Box.
+      Connect the screen with an HDMI cable on the side of the :abbr:`IoT (Internet of Things)` box.
+      :ref:`See the Raspberry Pi Schema <iot/config/connect_schema>`.
 
 .. important::
-   Screen(s) should be connected before the :abbr:`IoT (Internet of Things)` Box is switched on. If
+   Screen(s) should be connected before the :abbr:`IoT (Internet of Things)` box is switched on. If
    it is already on, connect the screen(s), and then restart the :abbr:`IoT (Internet of Things)`
-   Box by unplugging it and plugging it back into its power source.
+   box by unplugging it for ten seconds and plugging it back into its power source.
 
 .. warning::
    The usage of HDMI/Micro-HDMI adapters may cause issues which will result in a blank, black screen
```

Comment:
This is confusing since we're instructing people to use HDMI/micro-HDMI in the first place...what is meant that "Cables" should be used? Aren't HDMI cables...cables? 😅